### DISCUSSION TITLE
go2rtc integration

### DISCUSSION THREAD

**Discussion by Tokn59** [blakeblackshear/frigate#6035] [General]
In the doc I read "Use of the bundled go2rtc is optional. ". Does this mean I no longer have to add go2rtc as an add-on ? I just include the go2rtc camera configs under secton go2rtc: right ?

Thanks for clarifying this !

**Comment by blakeblackshear** [comment]
Frigate includes go2rtc directly in it's own container. You can include your go2rtc config within frigate's config. There is no need to run go2rtc separately, so you can remove the existing addon.

**Comment by Tokn59** [comment]
Perfect ! Thank You !!!!
